### DISCUSSION TITLE
How to Wrap 2D Map With plate-carre projection As It's Panned

### DISCUSSION THREAD

**Discussion by clinically-proven** [pelicanmapping/osgearth#2376] [Support Q&A] [answered]
Hi, is it possible to wrap a 2D map with plate-carre projection as it's panned across lat/lon boundaries?  This Cesium demo, when selected in 2D mode, is a good example of what I'd like to accomplish.  Any guidance would be appreciated.

https://sandcastle.cesium.com/
![2dwrap](https://github.com/gwaldron/osgearth/assets/150490651/9ba91bee-8f2d-408d-9aa4-efcbdd688d8c)

**Comment by gwaldron** [comment, marked as answer]
A couple leads on this.

There is an example here that uses Transforms to draw the MapNode multiple times for a scrolling effect:
https://github.com/gwaldron/osgearth/blob/master/src/applications/osgearth_infinitescroll/osgearth_infinitescroll.cpp

And here is an experimental approach that achieves the same thing but using multiple Views:
https://gist.github.com/gwaldron/bc7d57cbdd6b93c515aae026ab605384

Good luck :)

**Reply by clinically-proven** [reply]
Thanks for the quick reply, this is great stuff and is really helpful.
